noir_stdlib/src/sha256.nr

```diff
@@ -6,7 +6,7 @@ fn rotr32(a: u32, b: u32) -> u32 // 32-bit right rotation
 {
     // None of the bits overlap between `(a >> b)` and `(a << (32 - b))`
     // Addition is then equivalent to OR, with fewer constraints.
-    (a >> b) + (crate::wrapping_shift_left(a, 32 - b))
```

Comment:
can we remove `wrapping_shift_left` and `wrapping_shift_right` ?

Reply:
Done